docs/docs/reference/env-vars.mdx

```diff
@@ -0,0 +1,144 @@
+---
```

Comment:
Ah I think this file is an artifact from the rebase -- see comment about duplicated sections.